src/routes/example.js

```diff
@@ -0,0 +1,9 @@
+const { Router } = require("express");
```

Comment:
This example is not needed, but maybe we can rename it to "health check" and use to check if the API is ok